src/editor/FrequencySidebar.js

```diff
@@ -94,38 +74,47 @@ const FrequencySidebar = ( {
 					}
 
 					if ( 'preset_1' === value ) {
-						metaToUpdate.frequency_max = 5;
-						metaToUpdate.frequency_start = 3;
+						metaToUpdate.frequency_max = 0;
+						metaToUpdate.frequency_start = 0;
 						metaToUpdate.frequency_between = 3;
-						metaToUpdate.frequency_reset = 'month';
+						metaToUpdate.frequency_reset = 'day';
 					}
 
 					onMetaFieldChange( metaToUpdate );
 				} }
-				options={ getFrequencyOptions( isOverlay ) }
+				options={ frequencyOptions }
 			/>
 			{ 'custom' === frequency && (
 				<>
-					<NumberControl
-						className="newspack-popups__frequency-number-control"
-						label={ __( 'Max number of displays', 'newspack-popups' ) }
-						value={ frequency_max }
-						min={ 0 }
-						onChange={ value => onMetaFieldChange( { frequency_max: value } ) }
+					<ToggleControl
+						className="newspack-popups__frequency-toggle-control"
+						label={ __( 'Limit number of displays', 'newspack-popups' ) }
+						checked={ 0 < frequency_max }
+						onChange={ value => onMetaFieldChange( { frequency_max: value ? 1 : 0 } ) }
 					/>
+					{ 0 < frequency_max && (
+						<NumberControl
+							className="newspack-popups__frequency-number-control"
+							disabled={ 0 === frequency_max }
+							label={ __( 'Max number of displays', 'newspack-popups' ) }
```

Comment:
Suggestion: put this setting after the "start" and "every" settings. This would follow the logical flow of the interaction with a prompt:

<img width="271" alt="image" src="https://user-images.githubusercontent.com/7383192/190980340-e4a99839-58f1-4155-ab57-6c8601dacf77.png">

Reply:
Good call—updated in 1f0edcd.